build-support/bin/pre-commit.sh

```diff
@@ -19,7 +19,7 @@ $(git rev-parse --verify master > /dev/null 2>&1)
 if [[ $? -eq 0 ]]; then
   echo "* Checking imports" && ./build-support/bin/isort.sh || \
     die "To fix import sort order, run \`\"$(pwd)/build-support/bin/isort.sh\" -f\`"
-  echo "* Checking lint" && ./pants -q --exclude-target-regexp='testprojects/.*' --changed-parent=master lint || exit 1
+  echo "* Checking lint" && ./pants --exclude-target-regexp='testprojects/.*' --changed-parent=master lint || exit 1
```

Comment:
This change is because without quieting, the hook will only report on the number of lint errors, and not where to find them.

Reply:
Please open a ticket about that and comment here. Fixing it should be a small tweak to the WorkUnit label for the pex checkers.